qds_sdk/qubole.py

```diff
@@ -18,6 +19,8 @@ class Qubole:
     configuration parameters for QDS.
     """
 
+    MIN_POLL_INTERVAL = 3
```

Comment:
@mindprince I was using 3 sec for my testing. Forgot to add a comment on friday for the same. What minimum polling interval should we use? 5 sec?